### ISSUE TITLE
Mumble crashes on Ubuntu 22.04 on Wayland when hovering over the window-decorations (minimize, maximize and close buttons)

### ISSUE THREAD

**Issue by sneakpaw** [mumble-voip/mumble#5930]
### Description

Mumble crashes on Ubuntu 22.04 on Wayland when hovering over the window-decorations (minimize, maximize and close buttons)

### Steps to reproduce

1. Was working fine for months, up until a few weeks ago
   * installed on Ubuntu 22.04 with Flatpak (after the deb-package wasn't working correctly)
   * Compositor is Wayland
   * On Radeon-GPU
2. Installed latest updates.
3. When moving the mouse over the window-decorations, Mumble crashes.

### Mumble version

1.4.287

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Relevant log output

```shell
Okt 15 13:36:05 XXX info.mumble.Mumble.desktop[2312]: <W>2022-10-15 13:36:05.742 Wayland does not support QWindow::requestActivate()
Okt 15 13:36:11 XXX kernel: show_signal_msg: 1 callbacks suppressed
Okt 15 13:36:11 XXX kernel: mumble[2312]: segfault at 3900000008 ip 00007f2e880924e2 sp 00007ffcccbd2e10 error 4 in libqgnomeplatformdecoration.so[7f2e88090000+5000]
Okt 15 13:36:11 XXX kernel: Code: ff ff 4c 89 e7 e8 4e e8 ff ff 48 83 b8 00 01 00 00 00 74 49 4c 89 e7 4c 8d 6d c8 e8 38 e8 ff ff 4c 89 ef 4c 8b b0 00 01 00 00 <49> 8b 06 48 c7 45 d0 00 00 00 00 48 8b 58 18 e8 aa e4 ff ff 4c 89
Okt 15 13:36:11 XXX systemd[1141]: app-flatpak-info.mumble.Mumble-2302.scope: Consumed 27.088s CPU time.
Okt 15 13:36:21 XXX systemd[1141]: Started Application launched by gnome-shell.
Okt 15 13:36:21 XXX systemd[1141]: Started app-flatpak-info.mumble.Mumble-5328.scope.
```

**Comment by Krzmbrzl** [comment]
Refs.:
- https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System/issues/288
- https://github.com/hrydgard/ppsspp/issues/13119
- https://github.com/obsproject/obs-studio/issues/4645
- https://github.com/devinacker/bsnes-plus/issues/277

However, I can't seem to find a reference of the mentioned function in our code base... Have you checked that this is not a Flatpak-specific issue?

**Comment by sneakpaw** [comment]
No, I haven't checked that. The error actually occurs at a friend's computer and the Debian-package didn't work at all. We had some kind of dependency-issues a few months ago and therefore switched to the Flatpak-Version.

**Comment by Krzmbrzl** [comment]
Ok. I have asked on our Matrix server, whether anyone can confirm this issue.

**Comment by Krzmbrzl** [comment]
Meanwhile, I had a look at the [documentation](https://doc.qt.io/qt-5/qwindow.html#requestActivate) and seeing the see-also functions listed there, I found
https://github.com/mumble-voip/mumble/blob/e0c2b37daa2aaf235f670cff6f569c47a3481e92/src/mumble/MainWindow.cpp#L3573
which might in turn cause a call to `requestActivate`.

However, I don't see this function being called simply by hovering over the window decorations. So I'm not sure whether this is the right hunch...

**Comment by Hartmnt** [comment]
I can not reproduce this on my test VM using wayland and Ubuntu 23.10 and the latest Mumble build.

In case you are still using Mumble, can you please report, if this is still an issue with the latest version? If yes, which desktop environment are you using? The default Ubuntu Gnome fork or KDE?

**Comment by sneakpaw** [comment]
I am using:
- Latest Ubuntu 24.04 with
- Latest Mumble available as snap-package (1.5.0)
- Nvidia-GPU with closed-source drivers, so X11 not Wayland

**Comment by sneakpaw** [comment]
After writing my response (which I deleted in the meanwhile), I noticed, that this issue is from 2022. I cannot reproduce the issue any more. I also don't have the same setup any more. So: fixed for me.
